### PR TITLE
cylc gui: handle missing task status when setting up right-click menu

### LINKED ISSUE #1740: cylc gui: traceback on rightclicking when shutting down a suite
User has reported traceback sometimes being thrown when right clicking on an item in the cylc gui as a suite is shutting down.

cylctraceback

### PR DESCRIPTION
**Pull request by oliver-sanders**
Closes #1740 

Haven't been able to duplicate circumstances locally.

@arjclark please review.

### CODE REVIEW

File: lib/cylc/gui/view_graph.py [hunk 1]
```diff
@@ -184,11 +184,16 @@ def right_click_menu(self, event, task_id, type='live task'):
         menu.append(ungroup_rec_item)
 
         if type == 'live task':
+
```
Comment (hjoliver) [inline lib/cylc/gui/view_graph.py]: Spurious blank line?

**Comment by hjoliver** [comment]
This looks fine to me.

**Comment by arjclark** [comment]
Looks good to me. Manual testing confirms as working. @oliver-sanders - please remove blank line and then I'll merge this in.

**Comment by oliver-sanders** [comment]
Unrelated diagram, uploading here for wiki article:

kernel

**Pull request merged**